lib/skeleton/license.rb

```diff
@@ -1,16 +1,5 @@
-require 'skeleton/model'
-
 module Skeleton
-  class License < Model
+  class License
```

Comment:
If you don't mind validating it somewhere else a hash works fine.

If you want to keep a strict set of properties a Struct might be better.

```ruby
License = Struct.new(:url, :name)
```